about.html

```diff
@@ -42,13 +46,15 @@
 
           <!-- main section start -->
     <section class="main-part">
-        <div class="container">
+        <div class="about-container">
             <article>
-                <h2>Hello! Global Health Summit 2015</h2>
-                <h1>WORLD HEALTH <br> SUMMIT 2023</h1>
+              <h2 class="about-title">"Hello Sharing World"</h2>
+              <h1>
+                 Global Commons <br>Creative Summit 2015
+              </h1>
                 <p>The health sector is constantly evolving, and new challenges continue to emerge. As such, it is important to develop a comprehensive health development plan that can guide efforts to improve public health outcomes. Such a plan should incorporate current best practices, leverage innovative technologies and approaches, and prioritize evidence-based interventions.
 
-                    One key element of a health development plan for the future is the need to address health inequalities. In many countries, certain populations, such as low-income and marginalized groups, experience poorer health outcomes compared to others. </p>
+                    One key element of a health development plan for the future is the need to address health inequalities. In many countries, certain populations, such as low-income and  </p>
                     <h5 class="venue">Please Contact : globalhealth@gmail.com <br>Mirpur national stadium, Dhaka Bangladesh. </h5>
             </article>
         </div>
```

Comment:
- Kindly, Your background, color padding, margin, and text alignment aren't matching with the given design on both mobile and desktop versions and both `home` and `about` pages, please fix it, to have a better UI 👍 

| Your design                                                             | Figma design                                         |
| --------------------------------------------------------------------------------| --------------------------------------------------------- |
|<img src="https://user-images.githubusercontent.com/52010789/227700665-1685f887-9ed8-45c3-a945-db82ea372a3e.png" width="200">|<img src="https://user-images.githubusercontent.com/52010789/227700667-c4334f19-3c4a-4012-af92-4c0cfa669b01.png" width="200">|


😎️
#### Figma Design 👇
![2023-03-25_09-17](https://user-images.githubusercontent.com/52010789/227700669-6109bf71-f714-40b3-b867-9686684c30dc.png)
#### Given Design 👇
![2023-03-25_09-17_1](https://user-images.githubusercontent.com/52010789/227700885-3a00c595-2fc7-4959-967c-1d3f113a9ed1.png)